app/Controllers/Http/EventsController.ts

```diff
@@ -30,9 +31,15 @@ export default class EventsController {
 
   // post one event
   public async store({ request, response }: HttpContextContract) {
-    const body = request.only(this.eventFields)
-    const event = await Event.create(body)
-    response.json({ success: true, event })
+    try {
+      const payload = await request.validate(NewEventValidator)
+      // TODO set is_public if user.role equal moderator or admin
+      const body = Object.assign(payload, { is_public: false })
```

Comment:
der controller muss ja entscheiden ob das public ist oder nicht je nachdem ob ein moderator angemeldet ist. gibt es da noch einen anderen weg denn payload zu updaten?

Reply:
Die Datenbank setzt dieses Feld per default auf `false`. Du solltest es also nicht auch im Controller setzen müssen.
Später, wenn wir Rollen haben, können wir das Feld abhängig von der Rolle setzen. 

(Wir sollten außerdem versuchen in unseren PRs beim Thema zu bleiben :) )